lib/simplecov/defaults.rb

```diff
@@ -47,7 +47,7 @@
   # If we are in a different process than called start, don't interfere.
   next if SimpleCov.pid != Process.pid
 
-  if $ERROR_INFO # was an exception thrown?
+  if $! # was an exception thrown?
```

Comment:
@bf4 Thank you for the hint! I wasn't aware of the other `$ERROR_INFO` here right after the first emergence :astonished:
And changing all `$ERROR_INFO`s to `$!` made the build green finally!